FBAllocationTracker/NSObject+FBAllocationTracker.h

```diff
@@ -31,3 +31,12 @@
 @end
 
 #endif // _INTERNAL_FBAT_ENABLED
```

Comment:
Oh you are right, I've got confused.

I'm also thinking one last thing - to move them out of this category related files. I'd just put them in `Generations/FBAllocationTrackerZombieSupport.{h,mm}`. But that's just a nit, otherwise it looks good.